.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 
 # Matches all files in the .github/ dir, recursively.
-.github/ @spencerdrak @cgdolan @aryx @bmahe @apt-itude
+.github/ @aryx
```

Comment:
Why not just remove them altogether?

Reply:
I still want to review those. I do review them.

Reply:
I can remove the file